### PR TITLE
Added first cluster of JSON sensor files(40 files).

### PR DESCRIPTION
**Pull request by Msgtfrank**
Added this first cluster of 40 JSON files

### CODE REVIEW

File: src/apds9002/apds9002.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                    "unit": "ms",
+                    "typ" : 0.8,
+                    "max" : 2
+                }
+            },
+            "Platforms": {
+                "Intel Edison": {
+                    "Notes": ["Might require Seeed breakout board"]
+                },
+                "Arduinon 101": {
```
Comment (pylbert) [inline src/apds9002/apds9002.json]: Spelling - Arduino x 43

Comment (Msgtfrank) [inline src/apds9002/apds9002.json]: Oh wow I didn't notice that till now. It's even in the templete we've been using, which is based on another sensor. I'm a bit new to being a contributor in this, should I change the files and submit a new pull request or?...

Comment (pylbert) [inline src/apds9002/apds9002.json]: One way is to fix the spelling, do a commit --amend, and then force push.

Comment (Msgtfrank) [inline src/apds9002/apds9002.json]: Aha! Alright. I'll get that done. do I need to do another pull request to do that, or will the push apply to this request?

**Comment by dnoliver** [comment]
I think we should put some level of validation to those packages. We could use [jsonlint](https://github.com/zaach/jsonlint), and if we have well defined schema (i.e. every json must have the "Library" ) we can code an automated test to validate that for each of them.

File: src/apds9002/apds9002.json (hunk 1 above)
Comment (Propanu) [inline src/apds9002/apds9002.json]: When you update your branch it will automatically update the PR too.

**Pull request closed without merging**